### ISSUE TITLE
手写数字的行识别，模型不收敛，越训练loss越大

### ISSUE THREAD

**Issue by mymynew** [A-bone1/Attention-ocr-Chinese-Version#20]
训练数据集为手写数字的行识别：
![image](https://user-images.githubusercontent.com/32757878/42545816-b5dd0ec6-84ec-11e8-9d39-8672c2e7190a.png)
https://pan.baidu.com/s/1VuwANOFG-9xZKxlPgE5NoA
tfrecord生成用的是FSNS-tfrecord-generate中的generate_tfrecord_JPG.py因为原始图片为单通道图，所以加了个.convert('RGB')
现在的情况是，越训练loss越大：
2018-07-10 21:53:03,294tf_logging.py INFO global step 100: loss = 239.5235 (0.493 sec/step)
2018-07-10 21:53:53,060tf_logging.py INFO global step 200: loss = 558.6710 (0.474 sec/step)
2018-07-10 21:54:01,557tf_logging.py INFO Recording summary at step 216.
2018-07-10 21:54:02,394tf_logging.py INFO global_step/sec: 1.84779
2018-07-10 21:54:42,837tf_logging.py INFO global step 300: loss = 699.3590 (0.487 sec/step)
2018-07-10 21:55:32,055tf_logging.py INFO global step 400: loss = 965.1036 (0.495 sec/step)
2018-07-10 21:56:01,369tf_logging.py INFO Recording summary at step 458.
2018-07-10 21:56:02,398tf_logging.py INFO global_step/sec: 2.0166
2018-07-10 21:56:21,898tf_logging.py INFO global step 500: loss = 2043.1371 (0.484 sec/step)
2018-07-10 21:57:12,395tf_logging.py INFO global step 600: loss = 1464.7139 (0.484 sec/step)
2018-07-10 21:58:01,521tf_logging.py INFO Recording summary at step 697.
2018-07-10 21:58:02,426tf_logging.py INFO global_step/sec: 1.98286
2018-07-10 21:58:03,066tf_logging.py INFO global step 700: loss = 2526.5117 (0.515 sec/step)
2018-07-10 21:58:53,065tf_logging.py INFO global step 800: loss = 2093.2444 (0.499 sec/step)
2018-07-10 21:59:43,422tf_logging.py INFO global step 900: loss = 3336.6711 (0.499 sec/step)
2018-07-10 22:00:01,613tf_logging.py INFO Recording summary at step 935.
2018-07-10 22:00:02,409tf_logging.py INFO global_step/sec: 1.98362
2018-07-10 22:00:34,249tf_logging.py INFO global step 1000: loss = 3454.0398 (0.530 sec/step)
2018-07-10 22:01:24,575tf_logging.py INFO global step 1100: loss = 4373.2217 (0.546 sec/step)
![image](https://user-images.githubusercontent.com/32757878/42545765-857fa4c8-84ec-11e8-9e03-105108497beb.png)
可以帮我分析下吗？我找了一整天，已经能试的都试过了

**Comment by A-bone1** [comment]
@mymynew 可以看下#14的回答，看是不是数据集出现了问题。可以从tensorboard的text中检查下数据集的标签是否正确，长度是否匹配等

**Comment by mymynew** [comment]
@A-bone1 
我一张图片上，只有1个字符，跟我一张图片上有20个字符，他们都会被缩放到600*150。
这种情况下1个字符的会变得很大很大，这种情况下，有影响吗？

**Comment by A-bone1** [comment]
@mymynew 如果缩放后的图片产生了失真，是会有一定影响的
